tests/ErrorManagerTest.php

```diff
@@ -18,8 +20,10 @@ public function testGetErrors() : void
             ->willReturn(false);
 
         $errorManager = new ErrorManager($configuration);
+        ob_start();
```

Comment:
What is going on here?

Reply:
When `isAbortOnError` is false, `ErrorManager` echos the errors... which literally means that content was being echo'ed during the test. The `ErrorManager` needs to be more awesome. This was just a way to silence the output in a test, but it *is* unrelated to this PR.